sklearn_crfsuite/estimator.py

```diff
@@ -263,12 +260,14 @@ def __init__(self,
         self.averaging = averaging
         self.variance = variance
         self.gamma = gamma
+        self.model_filename = model_filename
+        self.keep_tempfiles = keep_tempfiles
```

Comment:
Why is this needed?

Reply:
https://github.com/TeamHG-Memex/sklearn-crfsuite/pull/74